styles.css

```diff
@@ -1,25 +1,87 @@
+/*-------------------------------------------------------------------
+						MACRO STYLING
+-------------------------------------------------------------------*/
+
 * {
 	box-sizing: border-box;
 }
 
+/* Don't fear I have added in my own tab focus styling */
+
+*:focus {
+	outline: none;
+}
+
 html {
 	font-size: 22px;
 	font-family: 'Orbitron', sans-serif;
 	font-weight: 600;
 }
 
+canvas {
+	image-rendering: crisp-edges;
+}
+
 body {
-	background-color: #120458;
 	color: #FE75FE;
 	margin-top: 60px;
 	background-image: url(images/BG.png);
 	background-repeat: no-repeat;
 	background-size: cover;
 	image-rendering: pixelated;
+	image-rendering: -moz-crisp-edges;
 	background-position: center;
 	background-attachment: fixed;
 }
 
+a, h2, p, h1 {
+	margin: 10px;
+	padding: 5px;
+}
+
+a, h2 {
+	font-size: 1rem;
+	align-content: center;
```

Comment:
Not sure this does anything for a `<a>` element. But could be wrong and it's doing something somewhere! so hard to tell!